src/components/IconButton/__stories__/iconButton.stories.module.scss

```diff
@@ -31,23 +31,23 @@
   }
 
   &RestoreButton {
-    margin-inline-end: var(--spacing-large);
+    margin-inline-end: var(--sb-spacing-large);
   }
 }
 
 .dashboard {
-  border: 1px solid var(--layout-border-color);
-  border-radius: var(--border-radius-medium);
+  border: 1px solid var(--sb-layout-border-color);
+  border-radius: var(--sb-border-radius-medium);
 
   &Header {
-    padding-inline-start: var(--spacing-small);
+    padding-inline-start: var(--sb-spacing-small);
     height: 49px;
     width: 100%;
-    border-bottom: 1px solid var(--layout-border-color);
+    border-bottom: 1px solid var(--sb-layout-border-color);
   }
   &Content {
     height: 200px;
     width: 100%;
-    background-color: var(--grey-background-color);
+    background-color: var(--sb-primary-background-hover-color);
```

Comment:
changed from a regular color variant to hover color variant
intentional?

Reply:
Changed it cause I didn't want to add new color token to the vibe-storybook-components just for this